packages/web-react/src/components/Collapse/useResizeHeight.ts

```diff
@@ -0,0 +1,30 @@
+import { useState, useEffect, MutableRefObject } from 'react';
```

Comment:
Please add a test for this if it is possible.

Reply:
Done